pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+u"""Wrapper for Python formatter (currently, ``black``) to update and to validate a repository.
+
+:copyright: Copyright (c) 2015 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkdebug import pkdp
+import pykern.pksubprocess
+import pykern.pkunit
+
+def run(path):
+    '''
```

Comment:
Black prefers double quotes. RS prefers text on one line (black doesn't care). Note the syntax of Args:

```py
"""Run black formatter on `path`

Args:
    path (object): string or py.path to file or directory
```

I used (object) here, because we don't care as long as it can be stringified.